js/style/style_batch.js

```diff
@@ -14,7 +14,8 @@ function styleBatch(style, work) {
 
     batch._style = style;
     batch._groupLayers = false;
-    batch._broadcastLayers = false;
+    batch._broadcastAllLayers = false;
+    batch._broadcastLayers = {};
```

Comment:
It's a little tricky that `this._broadcastLayers` and `this._style._broadcastLayers` both exist and are of different types